test/exporter/test_quantify_scheduler_exporter.py

```diff
@@ -42,21 +41,21 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
 
 class TestQuantifySchedulerExporter:
     def test_export(self):
-        register_manager = RegisterManager(QubitRegister(3), BitRegister(3))
-        ir = IR()
-        ir.add_gate(X(Qubit(0)))
-        ir.add_gate(CZ(Qubit(0), Qubit(1)))
-        ir.add_gate(Rz(Qubit(1), Float(2.34)))
-        ir.add_gate(Ry(Qubit(2), Float(1.23)))
-        ir.add_measurement(Measure(Qubit(0), Bit(0)))
-        ir.add_measurement(Measure(Qubit(1), Bit(1)))
-        ir.add_measurement(Measure(Qubit(2), Bit(2)))
+        builder = CircuitBuilder(3, 3)
+        builder.X(Qubit(0))
+        builder.CZ(Qubit(0), Qubit(1))
+        builder.Rz(Qubit(1), Float(2.34))
+        builder.Ry(Qubit(2), Float(1.23))
+        builder.measure(Qubit(0), Bit(0))
+        builder.measure(Qubit(1), Bit(1))
+        builder.measure(Qubit(2), Bit(2))
```

Comment:
Actually, you can already do what you are describing, since the `CircuitBuilder` returns `self` after each operation. So you can do:
```python
circuit = CircuitBuilder(3, 3).X(Qubit(0)).CZ(Qubit(0), Qubit(1)).Rx(Qubit(1), Float(2.34))...measure(Qubit(2), Bit(2)).to_circuit()
```
However, I personally prefer spreading it out over more lines, since I find it easier to read.

Reply:
On the other hand, it would be nice to be able to do
```python
builder = CircuitBuilder(3, 3)
builder.H([0,2])
```
which would be equal to
```python
builder = CircuitBuilder(3, 3)
builder.H(0)
builder.H(2)
```

Reply:
This is addressed in `CQT-59`

Reply:
`CQT-59` only speaks of measurements, not single qubit gates. Either there should be an extra ticket for that or it should be extended.

Reply:
I will extend the ticket